extensions/amp-story/1.0/test/validator-amp-story-amp-experiment.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-1.0.js"></script>
+  <title>My Story</title>
+  <meta name="description" content="Get started with amp-story">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link rel="canonical" href="grid-layer-templates.html">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+  <amp-story standalone title="Foo" publisher="AMP" publisher-logo-src="http://example.com/foo.png" poster-portrait-src="http://example.com/foo.png">
+    <amp-story-page id="fill-template-title">
+      <amp-story-grid-layer template="vertical">
+        <amp-experiment>
+            <script type="application/json">
+                {
+                  "background-color-test": {
+                    "variants": {
+                      "yellow": {
+                        "weight": 50,
+                        "mutations": [
+                          {
+                            "type": "attributes",
+                            "target": "h1",
+                            "attributeName": "style",
+                            "value": "background-color: #FFFF00"
+                          }
+                        ]
+                      },
+                      "red": {
+                        "weight": 50,
+                        "mutations": [
+                          {
+                            "type": "attributes",
+                            "target": "h1",
+                            "attributeName": "style",
+                            "value": "background-color: #FF0000"
+                          }
+                        ]
+                      }
+                    }
+                  }
+                }
+            </script>
+        </amp-experiment>
+
+        <!-- bare script invalid under amp-story-grid-layer -->
```

Comment:
We tend to label comments like these with the expected validation result first then the reason. E.g. `<-- Invalid: bare script not allowed as a child of amp-story-grid-layer -->`. Please update other comments in the file.

Reply:
Thanks for the heads up, fixed!

Reply:
Fixed